test/primitives-test.js

```diff
@@ -17,10 +17,10 @@ describe('Minim Primitives', function() {
       });
 
       it('should initialize the correct meta data', function() {
-        expect(el.meta.id.toValue()).to.equal('foobar');
-        expect(el.meta.class.toValue()).to.deep.equal(['a', 'b']);
-        expect(el.meta.title.toValue()).to.equal('Title');
-        expect(el.meta.description.toValue()).to.equal('Description');
+        expect(el.meta.get('id').toValue()).to.equal('foobar');
```

Comment:
So, I cannot do: `el.meta.id.toValue()` now. This seems extra verbose (shock me).